CardForm/src/test/java/com/braintreepayments/cardform/view/CardFormTest.java

```diff
@@ -534,7 +586,7 @@ public void getMobileNumber_returnsMobileNumber() {
 
     @Test
     public void setCardNumberError_setsError() {
```

Comment:
An additional test should be added for the card number error case similar to the other error focus tests. ie: `setCardNumberError_doesNotRequestFocusIfCardholderNameIsAlreadyFocused`